spdmlib/src/crypto/x509v3.rs

```diff
@@ -470,6 +470,60 @@ fn find_target_object_identifiers(data: &[u8], target_oid: &[u8]) -> SpdmResult<
     Ok(target_oid_find_success)
 }
 
+// IN  (extension sequences slice, target oid)
+// OUT true when find target oid
+// OUT false when not find target oid
+fn find_target_object_identifiers_in_extensions(
```

Comment:
`find_target_object_identifiers` can enter sequence slice or extnValue to find the target oid, so It's fine to keep using `find_target_object_identifiers` to search hardware oid, no need additional function to do the search work.

https://github.com/ccc-spdm-tools/spdm-rs/blob/cb06559c33c27231bad686e9a854ffc97710b50a/spdmlib/src/crypto/x509v3.rs#L461-L463